### ISSUE TITLE
modeling_causalvae  _training_step func  missing 2 required positional arguments: 'optimizer_idx' and 'global_step'

### ISSUE THREAD

**Issue by lxygoodjob** [PKU-YuanGroup/Open-Sora-Plan#249]
opensora/models/ae/videobase/causal_vae/modeling_causalvae.py
```
 def _training_step(self, batch, batch_idx):
        inputs = self.get_input(batch, "video")
        reconstructions, posterior = self(inputs)
        aeloss, log_dict_ae = self.loss(
            inputs,
            reconstructions,
            posterior,
            split="train",
        )
        self.log(
            "aeloss",
            aeloss,
            prog_bar=True,
            logger=True,
            on_step=True,
            on_epoch=True,
        )
        self.log_dict(
            log_dict_ae, prog_bar=False, logger=True, on_step=True, on_epoch=False
        )
        return aeloss
```
self.loss function missing 2 required positional arguments: 'optimizer_idx' and 'global_step'

**Comment by lxygoodjob** [comment]
Encoder & Decoder function         
conv_out: Module = "CasualConv3d", --> CausalConv3d

**Comment by LinB203** [comment]
The 'optimizer_idx' and 'global_step' will apply in GAN loss.
See https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/main/opensora/models/ae/videobase/causal_vae/modeling_causalvae.py#L408-L412

**Comment by lxygoodjob** [comment]
got it, thanks
